airflow/providers/snowflake/operators/snowflake.py

```diff
@@ -359,3 +368,159 @@ def __init__(
         self.authenticator = authenticator
         self.session_parameters = session_parameters
         self.query_ids: list[str] = []
+
+
+class SnowflakeSqlApiOperator(SnowflakeOperator):
```

Comment:
This is not good.
`SnowflakeOperator` is deprecated. We should not extend it's functionality.

This class needs to be refactored around `SQLExecuteQueryOperator`

Reply:
For reference... followup PR https://github.com/apache/airflow/pull/31751